### PR TITLE
[Task5] Added Value Objects and Steps classes

### PR DESCRIPTION
**Pull request by Zhukova-Iryna**

### CODE REVIEW

File: src/main/java/ua/price/steps/AuthenticationSteps.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package ua.price.steps;
+
+import org.openqa.selenium.WebDriver;
+
+public class AuthenticationSteps extends BaseSteps{
```
Comment (DmitriyBondarenko) [inline src/main/java/ua/price/steps/AuthenticationSteps.java]: Code format, please

File: src/main/java/ua/price/steps/AuthenticationSteps.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        enterTextInTextField(mainPage.regPasswordField(), regPassword);
+        clickWebElement(mainPage.submitRegistration());
+    }
+
+    public String getAccountLinkName() {
+        return getTextFromElement(mainPage.userAccount());
+    }
+
+    public boolean isErrorMessageDisplayed() {
+        return mainPage.errorLabel().isDisplayed();
```
Comment (DmitriyBondarenko) [inline src/main/java/ua/price/steps/AuthenticationSteps.java]: Please, use appropriate wait method here to ensure that element is displayed. Here and below.

File: src/main/java/ua/price/steps/BaseSteps.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    public void enterTextInTextField(WebElement textField, String inputText) {
+        clickWebElement(textField);
+        textField.clear();
+        textField.sendKeys(inputText);
+    }
+
+    public String getTextFromElement(WebElement webElement) {
+        return webElement.getText();
```
Comment (DmitriyBondarenko) [inline src/main/java/ua/price/steps/BaseSteps.java]: Use some wait here, please

File: src/main/java/ua/price/steps/BaseSteps.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    public void clickWebElement(WebElement webElement) {
+        waitUtils.elementToBeClickable(webElement).click();
```
Comment (DmitriyBondarenko) [inline src/main/java/ua/price/steps/BaseSteps.java]: Please, modify your `elementToBeClickable()` method to `clickWhenReady()` method and click on WebElement inside the method

**Review by DmitriyBondarenko — CHANGES_REQUESTED** [review]
Fix comments, please

File: src/main/java/ua/price/steps/AuthenticationSteps.java (hunk 1 above)
Comment (Zhukova-Iryna) [reply src/main/java/ua/price/steps/AuthenticationSteps.java]: Done

File: src/main/java/ua/price/steps/BaseSteps.java (hunk 4 above)
Comment (Zhukova-Iryna) [reply src/main/java/ua/price/steps/BaseSteps.java]: Done

File: src/main/java/ua/price/steps/BaseSteps.java (hunk 3 above)
Comment (Zhukova-Iryna) [reply src/main/java/ua/price/steps/BaseSteps.java]: Done

File: src/main/java/ua/price/steps/AuthenticationSteps.java (hunk 2 above)
Comment (Zhukova-Iryna) [reply src/main/java/ua/price/steps/AuthenticationSteps.java]: Done

**Review by DmitriyBondarenko — APPROVED** [review]

**Pull request merged**